apps/app/pages/admin/lectures/index.tsx

```diff
@@ -216,6 +236,69 @@ function Lectures() {
                 </Descriptions.Item>
               </Descriptions>
             </Space>
+            <Modal
+              title={lecture.event?.title}
+              open={open}
+              onOk={handleOk}
+              onCancel={handleOk}
+            >
+              <Descriptions size="small" column={1} layout="horizontal">
+                <Descriptions.Item
+                  labelStyle={labelStyle}
+                  label={
+                    <span>
+                      <UserAddOutlined /> Ninja
+                    </span>
+                  }
+                  span={1}
+                >
+                  <Link href={`/profile/ninja/${lecture.ninja?.id}`}>
+                    {lecture.ninja?.first_name + " " + lecture.ninja?.last_name}
+                  </Link>
+                </Descriptions.Item>
+                <Descriptions.Item
+                  labelStyle={labelStyle}
+                  label={
+                    <span>
+                      <UserOutlined /> Mentor
+                    </span>
+                  }
+                >
+                  <Link href={`/profile/mentor/${lecture.mentor?.id}`}>
+                    {lecture.mentor?.first_name +
+                      " " +
+                      lecture.mentor?.last_name}
+                  </Link>
+                </Descriptions.Item>
+                <Descriptions.Item
+                  labelStyle={labelStyle}
+                  label={
+                    <span>
+                      <EnvironmentOutlined /> Localização
+                    </span>
+                  }
+                  span={1}
+                >
+                  {selectedLocation?.address}
+                </Descriptions.Item>
+                <Descriptions.Item
+                  labelStyle={labelStyle}
+                  label={<span>Notas</span>}
+                  span={1}
+                >
+                  {lecture?.notes == null ? "Não há notas" : lecture?.notes}
```

Comment:
Since notes is an optional field, if they don't exist, simply just don't show this `Descriptions.Item`.